packit/cli/copr_build.py

```diff
@@ -57,6 +58,27 @@
     help="Installation instructions for the project to build in.",
     default=None,
 )
+@click.option(
+    "--listed-on-hp",
```

Comment:
Could this be only `--listed`? I find the `-on-hp` suffix somewhat unnecessary and confusing: "hp" like "Hewlett Packard"?

Edit: okay, I see that this coming from the Copr API. Still confusing.

Reply:
> Edit: okay, I see that this coming from the Copr API. Still confusing.

Yes, I want to match their naming.

`--listed` is too broad to me. What about full `--listed-on-homepage`?

Reply:
Let's put it in some context and try some variations:

```
$ packit copr-build --listed
$ packit copr-build --listed-on-hp
$ packit copr-build --listed-on-homepage
```

From the above it's somewhat difficult to tell what the `--listed*` flags do refer to. How about:

```
$ packit copr-build --list-project
```

Reply:
```
$ packit copr-build --list-project
```

I would expect a list of projects in the output..;)

If we want to be explicit, I would pick `--listed-on-homepage` or `--listed-on-hp`. We also have a description in the `--help`.

Reply:
`--list-on-homepage` to be consistent with `--preserve-project`